### PR TITLE
Viewer: ga event for drag and drop or file upload new report

### PR DESCRIPTION
**Pull request by ebidel**
R: all

Ga event when users drag and drop a new report to view or hit the file input.

### CODE REVIEW

File: lighthouse-viewer/app/src/lighthouse-report-viewer.js [hunk 1]
```diff
@@ -183,6 +183,8 @@ class LighthouseViewerReport {
       this._isNewReport = true;
 
       this.replaceReportHTML(JSON.parse(str));
+
+      ga('send', 'event', 'report', 'upload');
```
Comment (brendankenny) [inline lighthouse-viewer/app/src/lighthouse-report-viewer.js]: `upload` is a confusing choice of word in this function :) Which upload is this? Just for drag and drop on the page?

Comment (ebidel) [inline lighthouse-viewer/app/src/lighthouse-report-viewer.js]: There's a hidden file upload on the page as well as drag and drop. They both go through this callback.

 I originally used "view", but wanted to distinguish from the paste operation.

**Comment by ebidel** [comment]
I think this better. Any the report is changed, we hit a "view" event.

**Comment by ebidel** [comment]
PTAL

**Pull request merged**